spec/lib/submission_builder/ty2024/states/nj/documents/nj1040_spec.rb

```diff
@@ -32,6 +32,19 @@
         expect(xml.at("Exemptions DomesticPartnerRegular")).to eq(nil)
       end
 
+      it "populates line 8 XML fields" do
+        expect(xml.at("Exemptions YouBlindOrDisabled")).to eq(nil)
+        expect(xml.at("Exemptions SpouseCuPartnerBlindOrDisabled")).to eq(nil)
+      end
+
+      context "when filer is blind" do
+        let(:intake) { create(:state_file_nj_intake, :primary_blind) }
+        it "populates line 8 XML fields" do
```

Comment:
[pebble] for this and other tests in this file, can we update the test name to reflect the behavior? Since they all have the same exact test name, it doesn't give me any clues to what behavior to expect from reading it. Something more specific to each situation like "it populates line 8 XML for self" or similar might make sense